awxkit/awxkit/api/pages/api.py

```diff
@@ -355,6 +372,9 @@ def import_assets(self, data):
         self._assign_membership()
         self._assign_roles()
 
+        if self._has_error:
+            raise SystemExit(1)
```

Comment:
On second thought, we probably don't want to use an exception type that doesn't derive from `Exception` here.  That would bypass most try/except blocks by consumers of the library.

Reply:
Doing a simple `raise Exception("oops!")` at the top of `export_assets`, I'm able to trace that `awxkit/cli/__init__.py` line 25 is covered by a try/except that ultimately does a `sys.exit(1)`.

```
Traceback (most recent call last):
  File "/home/jrb/ansible/awx/awxkit/awxkit/cli/__init__.py", line 25, in run
    cli.parse_resource()
  File "/home/jrb/ansible/awx/awxkit/awxkit/cli/client.py", line 152, in parse_resource
    self.resource = parse_resource(self, skip_deprecated=skip_deprecated)
  File "/home/jrb/ansible/awx/awxkit/awxkit/cli/resource.py", line 212, in parse_resource
    response = command.handle(client, parser)
  File "/home/jrb/ansible/awx/awxkit/awxkit/cli/resource.py", line 177, in handle
    return client.v2.export_assets(**kwargs)
  File "/home/jrb/ansible/awx/awxkit/awxkit/api/pages/api.py", line 198, in export_assets
    raise Exception("oops!")
```

One problem I do see making the change here to just an `Exception`, the awx collection will exit early without doing output if we do this.  I might have steered you wrong.
:disappointed: 

@john-westcott-iv got any opinions about this?